### PR TITLE
Updating contribution file

### PR DESCRIPTION
**Pull request by mapedraza**
Improving contribution guidelines for developers to align with CII Best Practices requirements

### CODE REVIEW

**Comment by fgalan** [comment]
It seems there are some problems with the "Lint Markdown" action. See GitAction logs for details.

**Comment by mapedraza** [comment]
@fgalan Now it is fixed, It is ready to check and merge

File: docs/contribution.md [hunk 1]
```diff
@@ -1,6 +1,65 @@
...
+
+If you don't see your idea listed, and you think it fits into the goals of this guide, do one of the following:
+
+-   **If your contribution is minor,** such as a typo fix, open a pull request.
+-   **If your contribution is major,** such as a new guide, start by opening an issue first. That way, other people can
+    weigh in on the discussion before you do any work.
+
+### Pull Request protocol
+
+As explained in ([FIWARE Development Guidelines](https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Developer_Guidelines))
```
Comment (fgalan) [inline docs/contribution.md:31]: Similar comment done in a "twin" PR. What to do with this broken link?

Comment (mapedraza) [reply docs/contribution.md:31]: Decided to use this link while finding the fixed link / another alternative

**Review by fgalan — APPROVED** [review]
LGTM

File: docs/contribution.md (hunk 1 above)
Comment (fgalan) [reply docs/contribution.md:31]: Ok. NTC

**Pull request merged**